### DISCUSSION TITLE
Information about population on PK-Sim.

### DISCUSSION THREAD

**Discussion by adinizbe** [Open-Systems-Pharmacology/Forum#1272] [Questions & Problems]
Hi, I am Andrea and I would like to know if the regular population available (like Asian, Japanese, preterm ...) are already qualified (validated). Are there documents/publications demonstrating these validations?
Thanks

**Comment by StephanSchaller** [comment]
Hi Andrea, e.g., the original publication on preterms can be found here:
https://www.ncbi.nlm.nih.gov/pubmed/26323410

You can find many more evaluations in the publication content if you search for the keywords "preterm", "Japanese", ... here: 
https://github.com/Open-Systems-Pharmacology/OSP-based-publications-and-content/issues?q=is%3Aopen+is%3Aissue+label%3AJournal

**Reply by adinizbe** [reply]
Thank you very much for your answer
